airbyte_cdk/test/standard_tests/declarative_sources.py

```diff
@@ -64,7 +64,7 @@ def components_py_path(cls) -> Path | None:
     @classmethod
     def create_connector(
         cls,
-        scenario: ConnectorTestScenario,
+        scenario: ConnectorTestScenario | None,
```

Comment:
The scenario can now accept `None` but it does not default to `None`. For now at least, this prevents us from forgetting to provide `scenario`, while still allowing us to do so explicitly when needed.